### DISCUSSION TITLE
Alarm items showing unavailable

### DISCUSSION THREAD

**Discussion by rockwig** [tsightler/ring-mqtt#563] [Support Q&A]
This started yesterday morning, all of my Alarm devices (motion sensor, door sensors, CO listener, water sensors), are showing unavailable. I've looked through the logs and I don't see any errors. The Door Bell and video feed are working as expected. Anyone have any suggestions. I've restarted both HA and MQTT, and reloaded the config.

**Comment by rockwig** [comment]
A hard reboot of the box fixed it.

**Comment by aborruso** [comment]
Ciao Girolamo,
la tua tabella semplifica e mi piace. La fascia di età l'ho proposta perché in termini di privacy funziona meglio.
Ma possiamo provare come dici.

La PEC la possiamo inviare noi, perché alle volte chi riceve è più "toccato" da un indirizzo di un'organizzazione.
Quando dici che esistono già dei dati anonimizzati, hai qualche riferimento? Perché sarebbe utile citarlo nella richiesta.

Grazie
